src/Spryker/Zed/EventBehavior/Persistence/Propel/Behavior/EventBehavior.php

```diff
@@ -493,6 +504,102 @@ protected function isEventColumnsModified()
         ";
     }
 
+    /**
+     * @return string
+     */
+    protected function addGetOriginalValuesMethod()
+    {
+        $tableName = $this->getTable()->getName();
+        $originalValueColumns = $this->getKeepOriginalValueColumns();
+        $implodedOriginalValueColumns = implode("\n", array_map(function ($columnName) {
```

Comment:
`implodedOriginalValues` is enough

Reply:
It's should be `implodedOriginalValueColumnNames` actually.